conftest.py

```diff
@@ -16,8 +11,3 @@ def pytest_addoption(parser):
 def pytest_runtest_setup(item):
     if 'slow' in item.keywords and not item.config.getoption("--runslow"):
         pytest.skip("need --runslow option to run")
-
-
-def pytest_ignore_collect(path, config):
-    path = str(path)
-    return 'run_test.py' in path or path in skip_doctests
```

Comment:
I guess the `run_test.py` part was only needed for the conda recipe ( https://github.com/dask/dask/pull/449 ) that no longer exists ( https://github.com/dask/dask/pull/1162 ), correct?

Other than that validation question, agree that `collect_ignore` is a much cleaner way to go than this.

Reply:
Yes, and even then that was only needed if you ran `py.test` instead of `py.test dask`.